packages/data-context/src/util/array.ts

```diff
@@ -0,0 +1 @@
+export const toArray = (val?: string | string[]) => val ? typeof val === 'string' ? [val] : val : []
```

Comment:
You may be able to use lodash's [castArray](https://lodash.com/docs/#castArray) instead.

Reply:
Sure thing, good idea. Done: https://github.com/cypress-io/cypress/pull/25969/commits/4a275e49c68379d230fa77cf4006a1b60a93887f

Reply:
`[].concat()` does the same thing.


```
[].concat('foo') // ['foo']

[].concat(['foo', 'bar']) // ['foo', 'bar']
```

Lodash usage is fine too, of course.